infrastructure/modules/xc3/variables.tf

```diff
@@ -87,7 +87,7 @@ variable "hosted_zone_id" {
 variable "cognito_domain" {
   type        = string
   description = "Cognito Domain"
-  default     = "grafana-domain"
+  default     = "wajahat-domain"
```

Comment:
Can you dynamically add the namespace with the domain? where cognito_domain is being used.